torch/csrc/jit/mobile/function.cpp

```diff
@@ -41,6 +42,22 @@ bool Function::append_operator(const std::string& name,
   return true;
 }
 
+bool Function::append_builtin_function(const std::string& qualname) {
+  std::shared_ptr<torch::jit::Function> found_function;
+  auto qualname_ = c10::QualifiedName(qualname);
+  for (const auto& fn : customClassMethods()) {
```

Comment:
It is confusing that all builtin function calls here are actually methods. It is also not searching for builtin _methods_. If we also export method calls as CALL_INTERFACE then we wouldn't need to do this.